<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_13">
     <property name="title">
      <string>最近打开节目</string>
     </property>
     <addaction name="actionD_test_png"/>
    </widget>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="menu_13"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="separator"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="separator"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
    <addaction name="action_14"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_15"/>
    <addaction name="action_16"/>
    <addaction name="separator"/>
    <addaction name="action_20"/>
    <addaction name="separator"/>
    <addaction name="action_22"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>模式</string>
    </property>
    <widget class="QMenu" name="menu_14">
     <property name="title">
      <string>模式选择</string>
     </property>
     <addaction name="action_30"/>
     <addaction name="action_31"/>
    </widget>
    <widget class="QMenu" name="menu_15">
     <property name="title">
      <string>连接的备端</string>
     </property>
     <addaction name="action_32"/>
     <addaction name="separator"/>
     <addaction name="action_34"/>
    </widget>
    <widget class="QMenu" name="menu_16">
     <property name="title">
      <string>连接的主端</string>
     </property>
     <addaction name="action_35"/>
     <addaction name="separator"/>
     <addaction name="action_37"/>
    </widget>
    <widget class="QMenu" name="menu_17">
     <property name="title">
      <string>备端工程地址</string>
     </property>
     <addaction name="actionC_User_Project"/>
     <addaction name="separator"/>
     <addaction name="action_39"/>
    </widget>
    <widget class="QMenu" name="menu_18">
     <property name="title">
      <string>矩阵</string>
     </property>
     <addaction name="action_40"/>
     <addaction name="action_41"/>
    </widget>
    <widget class="QMenu" name="menu_19">
     <property name="title">
      <string>节目控制模式</string>
     </property>
     <addaction name="action_42"/>
     <addaction name="action_43"/>
    </widget>
    <addaction name="menu_14"/>
    <addaction name="menu_16"/>
    <addaction name="menu_15"/>
    <addaction name="menu_17"/>
    <addaction name="menu_18"/>
    <addaction name="action_27"/>
    <addaction name="separator"/>
    <addaction name="menu_19"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>舞台</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>舞台预览</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>媒体资源</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>节目</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_8">
    <property name="title">
     <string>时间线</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_9">
    <property name="title">
     <string>效果</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_10">
    <property name="title">
     <string>工具</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_11">
    <property name="title">
     <string>窗口</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_12">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menu_7"/>
   <addaction name="menu_8"/>
   <addaction name="menu_9"/>
   <addaction name="menu_10"/>
   <addaction name="menu_11"/>
   <addaction name="menu_12"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="nvresource.qrc">
     <normaloff>:/images/newFile.png</normaloff>:/images/newFile.png</iconset>
   </property>
   <property name="text">
    <string>新建工程</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="nvresource.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开工程</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="nvresource.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存工程</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>工程另存为</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="nvresource.qrc">
     <normaloff>:/images/package.png</normaloff>:/images/package.png</iconset>
   </property>
   <property name="text">
    <string>打包工程</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>关闭连接的计算机</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>重启连接的计算机</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>保存为默认布局</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionD_test_png">
   <property name="text">
    <string>D:/test.png</string>
   </property>
  </action>
  <action name="action_15">
   <property name="icon">
    <iconset resource="nvresource.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>退回</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_16">
   <property name="icon">
    <iconset resource="nvresource.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>重复上一条命令</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>选择所有</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>选择到最后</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_22">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>对齐</string>
   </property>
  </action>
  <action name="action_27">
   <property name="text">
    <string>备端允许更新</string>
   </property>
  </action>
  <action name="action_30">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>主端</string>
   </property>
  </action>
  <action name="action_31">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>备端</string>
   </property>
  </action>
  <action name="action_32">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>无备端连接</string>
   </property>
  </action>
  <action name="action_34">
   <property name="text">
    <string>选择已连接的备端</string>
   </property>
  </action>
  <action name="action_35">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>无主端连接</string>
   </property>
  </action>
  <action name="action_37">
   <property name="text">
    <string>选择已连接的主端</string>
   </property>
  </action>
  <action name="actionC_User_Project">
   <property name="text">
    <string>C:/User/Project</string>
   </property>
  </action>
  <action name="action_39">
   <property name="text">
    <string>选择工程地址</string>
   </property>
  </action>
  <action name="action_40">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_41">
   <property name="text">
    <string>切换</string>
   </property>
  </action>
  <action name="action_42">
   <property name="text">
    <string>播控模式</string>
   </property>
  </action>
  <action name="action_43">
   <property name="text">
    <string>预监模式</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="nvresource.qrc"/>
 </resources>
 <connections/>
</ui>
